ca/server.go

```diff
@@ -69,6 +69,26 @@ func (s *Server) SetReconciliationRetryInterval(reconciliationRetryInterval time
 	s.reconciliationRetryInterval = reconciliationRetryInterval
 }
 
+// GetUnlockKey is responsible for returning the current unlock key used for encrypting TLS private keys and
+// other at rest data.  Access to this RPC call should only be allowed via mutual TLS from managers.
+func (s *Server) GetUnlockKey(ctx context.Context, request *api.GetUnlockKeyRequest) (*api.GetUnlockKeyResponse, error) {
+	log.G(ctx).WithFields(logrus.Fields{
+		"method": "GetUnlockKey",
+	})
+
+	var managerUnlockKey []byte
+	s.store.View(func(tx store.ReadTx) {
+		cluster := store.GetCluster(tx, s.securityConfig.ClientTLSCreds.Organization())
+		if cluster.Spec.EncryptionConfig.AutoLockManagers {
+			managerUnlockKey = cluster.UnlockKeys.Manager
+		}
+	})
+
+	return &api.GetUnlockKeyResponse{
+		UnlockKey: managerUnlockKey,
```

Comment:
> But the proxies are managers anyway, which can have access to the unlock key anyway.

This is true now but may not be in the future. It would be good to reduce the exposure of that key so we don't have to patch these holes if we change the model.

Put another way, should this code have to know whether or not the key is available to all managers or just the target manager?